bundles/core/org.eclipse.smarthome.core.audio/src/main/java/org/eclipse/smarthome/core/audio/AudioManager.java

```diff
@@ -0,0 +1,460 @@
+        if (config != null) {
+            this.defaultSource = config.containsKey(CONFIG_DEFAULT_SOURCE)
+                    ? config.get(CONFIG_DEFAULT_SOURCE).toString() : null;
+            this.defaultSink = config.containsKey(CONFIG_DEFAULT_SINK) ? config.get(CONFIG_DEFAULT_SINK).toString()
+                    : null;
+        }
+    }
+
+    /**
+     * Plays the passed audio stream using the default audio sink.
+     *
+     * @param audioStream The audio stream to play
+     */
+    public void play(AudioStream audioStream) {
+        play(audioStream, null);
+    }
+
+    /**
+     * Plays the passed audio stream on the given sink.
+     *
+     * @param audioStream The filename to play
+     * @param sinkId The id of the audio sink to use or null
+     */
+    public void play(AudioStream audioStream, String sinkId) {
+        if (audioStream != null) {
+            AudioSink sink = null;
+            if (sinkId == null) {
+                sink = getSink();
+            } else {
+                sink = audioSinks.get(sinkId);
+            }
+
+            if (sink != null) {
+                try {
+                    sink.process(audioStream);
+                } catch (UnsupportedAudioFormatException e) {
+                    logger.error("Error playing '{}': {}", audioStream.toString(), e.getMessage());
+                }
+            } else {
+                logger.warn("Failed playing audio stream '{}' as no audio sink was found.", audioStream.toString());
+            }
+        }
+    }
+
+    /**
+     * Plays an audio file from the "sounds" folder using the default audio sink.
+     *
+     * @throws AudioException in case the file does not exist or cannot be opened
+     */
+    public void playFile(String fileName) throws AudioException {
+        playFile(fileName, null);
+    }
+
+    /**
+     * Plays an audio file from the "sounds" folder using the given audio sink.
+     *
+     * @throws AudioException in case the file does not exist or cannot be opened
+     */
+    public void playFile(String fileName, String sink) throws AudioException {
+        File file = new File(System.getProperty(ConfigConstants.CONFIG_DIR_PROG_ARGUMENT) + File.separator + SOUND_DIR
+                + File.separator + fileName);
+        FileAudioStream is = new FileAudioStream(file);
+        play(is, sink);
+    }
+
+    /**
+     * Stream audio from the passed url using the default audio sink.
+     *
+     * @throws AudioException in case the url stream cannot be opened
+     */
+    public void stream(String url) throws AudioException {
+        stream(url, null);
+    }
+
+    /**
+     * Stream audio from the passed url to the given sink
+     *
+     * @param url The url to stream from
+     * @param sinkId The id of the audio sink to use or null
+     * @throws AudioException in case the url stream cannot be opened
+     */
+    public void stream(String url, String sinkId) throws AudioException {
+
```

Comment:
As you already need to touch this file, I think this empty line could be removed.